clustergroup/templates/plumbing/applications.yaml

```diff
@@ -78,7 +78,7 @@ spec:
             - name: global.namespace
               value: {{ $.Values.global.namespace }}
             - name: clusterGroup.name
-              value: {{ .Values.clusterGroup.name }}
+              value: {{ $.Values.clusterGroup.name }}
```

Comment:
Spurious change?

Reply:
@mbaldessari In my tests this was giving me issues.  Perhaps we can talk about it this morning.